content/docs/apis.md

```diff
@@ -2,57 +2,176 @@
 title: APIs
 description: Interact with TiKV using the raw key-value API or the transactional key-value API
 weight: 4
-draft: true
+draft: false
 ---
 
 TiKV offers two APIs that you can use to interact with it:
 
-* A [raw](#raw) API
-* A [transactional](#transactional) API
+API | Description | Atomicity | Use when...
+:---|:------------|:----------|:-----------
+[Raw](#raw) | A lower-level key-value API for interacting directly with individual key-value pairs. | Single key | Your application doesn't require distributed transactions or multi-version concurrency control (MVCC)
+[Transactional](#transactional) | A higher-level key-value API that provides ACID semantics | Multiple keys | Your application requires distributed transactions and/or MVCC
+
+{{< info >}}
+It is **not recommended or supported** to use both the raw and transactional APIs on the same keyspace.
+{{< /info >}}
+There are several clients that connect to TiKV:
+
+* [Rust](https://github.com/tikv/client-rust)
+* [Java](https://github.com/tikv/client-java)
+* [Go](https://github.com/pingcap/tidb/store/tikv)
+
+Below we use the Rust client for some examples, but you should find all clients work similarly.
+
+## Connecting a Client
+
+This guide assumes you are using Rust and have Rust installed through `rustup` or other means.
+
+To start, open the `Cargo.toml` of your project and add the `tikv-client` as a dependency. You'll also need to depend on `futures`.
+
+<!-- TODO: Use crates.to once published -->
+
+```toml
+[dependencies]
```

Comment:
Maybe worth adding the `edition = "2018"` line

Reply:
Since we're saying we expect you're using Rust >=1.31 now it doesn't matter what edition they're using.

Reply:
I don't think that is right - you still have to opt-in to the 2018 edition. If they don't do that, they'll need `extern crate`s.